charts/egeria-platform/values.yaml

```diff
@@ -0,0 +1,117 @@
+# Egeria Settings
+egeria:
+  # Global Egeria Settings
+  logging: error
+  version: "3.5"
+  debug: true
+
+  # persistence
+  persistence: true
+  storageClass: ibmc-vpc-block-10iops-tier
+  storageSize: 8Gi
+  repositoryType: "local-graph-repository"
+
+  # communication
+  cohort: mds
+  viewOrg: org
+  kafkaEndpoint: egeria-kafka:9092
+
+  # userid used for admin ops (no security enabled by default)
+  user: garygeeke
+
+  # list of servers that should start on this platform
+  # the names must correspond to a config-<name>.sh script
+  # as well as to the actual instance name
+  serverList: mds1,view1
+
+  # OMAG Server
+  # usually you need to configure a connection to an OMAG server
+  # the values below will result in EGERIA_OMAG_SERVER_NAME and EGERIA_OMAG_SERVER_URL environment variables
+  # if platformUrl is empty this will point to itself
+  omagServer:
+    name: mds1
+    platformUrl: ""
+    
+
+# Additional environment variables that are needed for setup the servers
+extraEnv: {}
+
+# name a secret to mount as environment variables to store secrets
+secretName: ""
+
+replicaCount: 1
+
+image:
+  repository: quay.io/odpi/egeria
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "3.5-SNAPSHOT"
```

Comment:
I would be inclined to go with 3.4 unless you need function in 3.5-SNAPSHOT, though our other charts do use 3.5-SNAPSHOT as they're also used for testing.

But crucially we need to make the versions align for clarity
 * use 3.5.0-prelease.0 as an example of a 'prerelease chart' (use --devel to list) or '3.5.0' for a released chart (which this isn't yet, as more testing needed)
 * Keep the app version in the top level Chart.yaml to indicate the default version of egeria we use, whether that is '3.4' or '3.4-SNAPSHOT' - again helps with clarity

Reply:
See https://github.com/odpi/egeria-charts/pull/107#discussion_r777954776